src/ast.ts

```diff
@@ -144,10 +160,26 @@ export function reference(base: Exp, prop: string | Exp): ExpReference {
 // immutability of the original expression).
 export function copyExp(exp: Exp): Exp {
   exp = <Exp> util.extend({}, exp);
-  if (exp.type === 'op' || exp.type === 'call') {
-    (<ExpOp> exp).args = util.copyArray((<ExpOp> exp).args);
+  switch (exp.type) {
+  case 'op':
```

Comment:
I just checked - I do have that warning turned on - it does not file in this particular case of am empty case block "falling through" to a non-empty one.

http://eslint.org/docs/rules/no-fallthrough.html